addons/point_of_sale/models/pos_order.py

```diff
@@ -530,19 +189,40 @@ def _filtered_for_reconciliation(self):
         readonly=True,
         states={'draft': [('readonly', False)]},
     )
+    payment_ids = fields.One2many('pos.payment', 'pos_order_id', string='Payments', readonly=True)
+    session_move_id = fields.Many2one('account.move', string='Session Journal Entry', related='session_id.move_id', readonly=True, copy=False)
+    is_fully_paid = fields.Boolean('Is Fully Paid', compute='_compute_is_fully_paid')
```

Comment:
we should not add fields is_fully_paid and amount_to_pay on the model as it is only used in a wizard one time.

Remove those two fields and compute the difference where it is needed